utilities/DatabaseUtilities.js

```diff
@@ -1,92 +1,101 @@
-import mysql from 'mysql2/promise';
-import fs from 'fs';
-import path from 'path';
+import mysql from "mysql2/promise";
+import fs from "fs";
+import path from "path";
 
 let pool = null;
 
 export async function createSchema() {
-    const schemaPath = path.resolve('schema.sql');
-    if (!fs.existsSync(schemaPath)) throw new Error("Schema file not found!");
+  const schemaPath = path.resolve("schema.sql");
```

Comment:
I appreciate the use of double quotes instead of single quotes.

Double quotes GOAT

Reply:
Preach it, brother. Amen.